ia/src/parser.py

```
@@ -0,0 +1,41 @@
+import sys
+
+from utils import exit_error
+from models import iaParser
+
+def ia_parser(argv: list[str]):
```

Ruff (0.11.9)

`ia_parser` is too complex (11 > 10)

(C901)